enterprise_access/apps/subsidy_access_policy/mocks.py

```diff
@@ -0,0 +1,81 @@
+""" Mock clients for subsidy_access_policy models. """
+
+import mock
+
+
+class group_client():
+    """
+    API client for the groups service.
+    """
+    @classmethod
+    def group_contains_learner(cls, group_uuid, learner_id):
+        """Group service api"""
+        return mock.MagicMock(group_uuid, learner_id)
+
+    @classmethod
+    def get_groups_for_learner(cls, learner_id):
+        """Group service api"""
+        return mock.MagicMock(learner_id)
+
+class catalog_client():
+    """
+    API client for the enterprise-catalog service.
+    """
+    @classmethod
+    def catalog_contains_content(cls, catalog_uuid, content_key):
+        """Catalog service api"""
+        return mock.MagicMock(catalog_uuid, content_key)
+
+class subsidy_client():
+    """
+    API client for the groups service.
+    """
+    @classmethod
+    def is_redeemable(cls, subsidy_uuid, learner_id, content_key):
+        """Subsidy service api"""
+        return mock.MagicMock(subsidy_uuid, learner_id, content_key)
+
+    @classmethod
+    def redeem(cls, subsidy_uuid, learner_id, content_key):
```

Comment:
Do we want to change this to `create_transaction`? @iloveagent57 
So that is more inline with https://github.com/openedx/enterprise-subsidy/blob/180b1b3dd9e7dddb5c9c7f6175b424bae685a3a5/docs/decisions/0002-initial-api-specification.rst#post-enterprise-subsidy-transaction

Reply:
No